src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -163,7 +165,9 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
       }
       if (jetbrainsUserId != agentUserId) {
         if (agentUserId != null) {
-          logger.error("User id in JetBrains is different from agent")
+          logger.warn("User id in JetBrains is different from agent: restarting agent...")
```

Comment:
changed to warn (so a user does not see an internal error anymore) + restarting the agent (which should be a fix)